RelationalAI/Client.cs

```diff
@@ -50,13 +53,19 @@ public HttpClient HttpClient
             set { _rest.HttpClient = value; }
         }
 
+        public TraceSource TraceSource
```

Comment:
to research - right now we think we can only do file-based configuration in .NET framework, but we need it in .NET core, how should we resolve this so programmatic configuration is not required?

Possibilities:
  Our own config file interpretation
  XUnit's config file
  (understanding how this interplays with other environments the SDK is being run in such as ASP.NET server)

Reply:
People complains about the missing ability to configure diagnostics traces using app.config or a similar configuration file, https://github.com/dotnet/runtime/issues/23937
keep digging around, hopefully we won't need a custom config file interpretation :pray: